conans/client/installer.py

```diff
@@ -396,6 +396,8 @@ def _build(self, nodes_by_level, keep_build, root_node, graph_info, remotes, bui
                 self._propagate_info(node, using_build_profile)
                 if node.binary == BINARY_EDITABLE:
                     self._handle_node_editable(node, graph_info)
+                    # Need a temporary package revision for package_revision_mode
+                    node.prev = "0"
```

Comment:
`0` is a valid package revision, can we use any other? If we just need a value, `PREV_UNKNOWN`?

Can we initialize this member in the `node` class definition?

---

...but I don't know if that value has a meaning, I can see in the sources somewhere else:

```python
# It is requested to use, but not defined (binary not build yet)
self.package_revision = self.full_package_revision or PREV_UNKNOWN
```

Reply:
"0" is the revision when we have no revisions. It is not a valid one, it is a placeholder to mean that we have no revisions. I thought that for editables it made kind of sense, as we are not revisioning, but yes, I guess a PREV_UNKNOWN could be better.